package.json

```diff
@@ -21,21 +21,25 @@
   "author": "Josh Perez <josh.perez@airbnb.com>",
   "license": "MIT",
   "devDependencies": {
-    "@kadira/storybook": "^1.26.0",
-    "babel-cli": "^6.9.0",
-    "babel-istanbul": "^0.8.0",
-    "babel-preset-airbnb": "^1.0.1",
-    "casual": "^1.5.3",
-    "chai": "^3.4.1",
+    "@kadira/storybook": "2.4.x",
+    "babel-cli": "^6.14.0",
+    "babel-core": "^6.14.0",
+    "babel-istanbul": "^0.11.0",
+    "babel-preset-airbnb": "^2.0.0",
+    "casual": "^1.5.5",
+    "chai": "^3.5.0",
+    "chai-enzyme": "^0.5.1",
     "enzyme": "^1.3.1",
-    "eslint": "^1.8.0",
-    "eslint-config-airbnb": "^1.0.0",
-    "eslint-plugin-react": "^3.7.1",
-    "mocha": "^2.3.3",
+    "eslint": "^3.5.0",
+    "eslint-config-airbnb": "^11.1.0",
+    "eslint-plugin-import": "^2.0.0",
+    "eslint-plugin-jsx-a11y": "^2.2.2",
+    "eslint-plugin-react": "^6.2.2",
+    "mocha": "^3.0.2",
     "raw-loader": "^0.5.1",
-    "react": "^0.14.3",
-    "react-addons-test-utils": "^0.14.8",
-    "react-dom": "^0.14.3",
+    "react": "^15.3.1",
```

Comment:
we shouldn't update the dev deps here - we're still on 0.14.

Also, updating the dev deps should be done in a separate commit.